src/commands/subcommands/community/handleLeaderboard.ts

```diff
@@ -44,7 +44,7 @@ export const handleLeaderboard: CommandHandler = async (
       (level, index) => [index + 1, level.userTag, level.level, level.points]
     );
 
-    const authorLevel = mapped.find((el) => el[1] === interaction.user.tag);
+    const authorLevel = mapped.find((el) => el[1] === interaction.user.username);
```

Comment:
```suggestion
    const authorLevel = mapped.find((el) => new RegExp(`^${name}(?!#\d{4})?$`).test(el[1])));
```

Because the database will still have records that are `username#0000`, we need to find the correct username while accounting for the discriminator.